grails-app/services/com/unifina/service/SubscriptionService.groovy

```diff
@@ -14,6 +14,16 @@ import grails.compiler.GrailsCompileStatic
 class SubscriptionService {
 	PermissionService permissionService
 
+	void deleteProduct(Product product) {
```

Comment:
Can we have a more accurate name, e.g. `deleteAllSubscriptionsOfProduct(Product product)` or `deleteAllByProduct(Product product)` something along those lines.